tests/test_ecs.py

```diff
@@ -185,3 +186,79 @@ def test_container_instance_resource(self):
         resources = p.run()
         self.assertEqual(len(resources), 1)
 
+    def test_container_instance_update_agent(self):
+        session_factory = self.replay_flight_data('test_ecs_container_instance_update_agent')
+        p = self.load_policy(
+            {'name': 'container-instance-update-agent',
+             'resource': 'ecs-container-instance',
+             'actions':[
+                 {'type': 'update-agent'}
+                 ]},
+             session_factory=session_factory)
+        resources = p.run()
+        if self.recording:
+            time.sleep(60)
+        client = session_factory().client('ecs')
+        c_instances = client.list_container_instances(
```

Comment:
rather doing a list and describe, just do the describe with the resources containersArn